sdk/formrecognizer/Azure.AI.FormRecognizer/src/BoundingBox.cs

```diff
@@ -16,26 +17,28 @@ namespace Azure.AI.FormRecognizer.Models
     /// </summary>
     public readonly struct BoundingBox
     {
+        private readonly PointF[] _points;
```

Comment:
`BoundingBox` is a struct, so it has a public parameterless constructor that we cannot override. This constructor was forcibly setting `Points` to null, and we couldn't overwrite it later on since the struct is `readonly`.

I decided to create a private `_points` array to take this role. `Points` will be a property that wraps `_points` and returns an empty array instead of null.

Reply:
👍